.gitignore

```diff
@@ -1,3 +1,4 @@
 /target
 **/*.rs.bk
 Cargo.lock
+.vscode
```

Comment:
For this type of exceptions you should use global gitignore: https://stackoverflow.com/questions/7335420/global-git-ignore